.ci/build_pipeline_definitions

```diff
@@ -34,45 +34,21 @@ ops-toolbelt:
                 build: ~
 EOF
 
-for i in `cat "${SOURCE_PATH}"/.ci/k8s_versions`
+for kv_short in `cat "${SOURCE_PATH}"/.ci/k8s_versions |cut -d '.' -f 1,2`
 do
-    cat >> "${SOURCE_PATH}"/.ci/pipeline_definitions << EOF
-          ops-toolbelt-aws-k$i:
-            registry: 'gcr-readwrite'
-            image: 'eu.gcr.io/gardener-project/gardener/ops-toolbelt-aws-k$i'
-            dockerfile: ops-toolbelt-aws-k$i.dockerfile
-            inputs:
-              steps:
-                build: ~
-          ops-toolbelt-az-k$i:
-            registry: 'gcr-readwrite'
-            image: 'eu.gcr.io/gardener-project/gardener/ops-toolbelt-az-k$i'
-            dockerfile: ops-toolbelt-az-k$i.dockerfile
-            inputs:
-              steps:
-                build: ~
-          ops-toolbelt-gcp-k$i:
-            registry: 'gcr-readwrite'
-            image: 'eu.gcr.io/gardener-project/gardener/ops-toolbelt-gcp-k$i'
-            dockerfile: ops-toolbelt-gcp-k$i.dockerfile
-            inputs:
-              steps:
-                build: ~
-          ops-toolbelt-openstack-k$i:
-            registry: 'gcr-readwrite'
-            image: 'eu.gcr.io/gardener-project/gardener/ops-toolbelt-openstack-k$i'
-            dockerfile: ops-toolbelt-openstack-k$i.dockerfile
-            inputs:
-              steps:
-                build: ~
-          ops-toolbelt-aliyun-k$i:
-            registry: 'gcr-readwrite'
-            image: 'eu.gcr.io/gardener-project/gardener/ops-toolbelt-aliyun-k$i'
-            dockerfile: ops-toolbelt-aliyun-k$i.dockerfile
-            inputs:
-              steps:
-                build: ~
+    for iaas_provider in `cat "${SOURCE_PATH}"/.ci/iaas_providers`
+    do
+
+      cat >> "${SOURCE_PATH}"/.ci/pipeline_definitions << EOF
+            ops-toolbelt-$iaas_provider-k$kv_short:
+              registry: 'gcr-readwrite'
+              image: 'eu.gcr.io/gardener-project/gardener/ops-toolbelt-$iaas_provider-k$kv_short'
+              dockerfile: ops-toolbelt-$iaas_provider-k$kv_short.dockerfile
+              inputs:
+                steps:
+                  build: ~
```

Comment:
the indenting is not correct. After the fix, please re-run the script to fix `.ci/pipeline_definitions`
```suggestion
          ops-toolbelt-$iaas_provider-k$kv_short:
            registry: 'gcr-readwrite'
            image: 'eu.gcr.io/gardener-project/gardener/ops-toolbelt-$iaas_provider-k$kv_short'
            dockerfile: ops-toolbelt-$iaas_provider-k$kv_short.dockerfile
            inputs:
              steps:
                build: ~
```